### DISCUSSION TITLE
LightGBM

### DISCUSSION THREAD

**Discussion by TonciG** [biolab/orange3#6781] [Ideas]
Hi,
Are you planning to implement LightGBM in Orange in the near future and is there a reason why you have not done it already?
Thanks!
Tonci

**Comment by janezd** [comment]
We already have several implementations of gradient boosting, which cause various amounts of trouble (xgboost is mostly ok, and catboost seems problematic to install at least for combinations on macOs). Adding yet another boosting algorithm would add to this.

We generally avoid unnecessary dependencies, in particular if they are not in pure python.

**Reply by TonciG** [reply]
Thanks on your answer. Understand the potential challenges. It is a pity since LightGBM is very fast and offers good performance, sometimes even better than that of CatBoost and XGBoost.

**Comment by markotoplak** [comment]
We would welcome this addition as an add-on.

**Reply by TonciG** [reply]
Agree, it would be very good add-on. Thanks!
